src/ui/buttons/props.js

```diff
@@ -659,7 +661,8 @@ export function normalizeButtonProps(props : ?ButtonPropsInputs) : RenderButtonP
         applePaySupport = false,
         supportsPopups = false,
         supportedNativeBrowser = false,
-        experience = ''
+        experience = '',
+        showPayLabel
```

Comment:
Should this default to `false`?

Reply:
It should default to `true` (i.e. the non-PPoF flow). I did that [here](https://github.com/paypal/paypal-checkout-components/pull/2038/files#diff-a27dac7ef82b828a0bd903741902450634b92feaf7710393257a01066fac3363R317), but should it be higher up the component hierarchy?